test/unit/alignment/multiple/align_multiple_test.cpp

```diff
@@ -7,28 +7,42 @@
 
 #include <gtest/gtest.h>
 
+#include <meta/meta.hpp>
```

Comment:
Why do we need this? I haven't seen this before...
Maybe try to compile without it and see if it still works.

Reply:
No we don't and some more of these includes were not needed.   Thanks again !